nornir/core/inventory.py

```diff
@@ -405,16 +405,7 @@ def get_connection(self, connection: str, configuration: Config) -> Any:
                 platform=conn.platform,
                 extras=conn.extras,
             )
-        return self.connections[connection].connection
-
-    def get_connection_state(self, connection: str) -> Dict[str, Any]:
-        """
-        For an already established connection return its state.
-        """
-        if connection not in self.connections:
-            raise ConnectionNotOpen(connection)
-
-        return self.connections[connection].state
+        return self.connections[connection]
```

Comment:
This is the line that changed (pertaining to returning the NAPALM/Netmiko object (i.e. used to be `self.connections[connection].connection`)